### ISSUE TITLE
Sass maps complain about indentation

### ISSUE THREAD

**Issue by alan-agius4** [stylelint/stylelint#1034]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

"A bug where indentation is fired when it's not expected"

> Which rule, if any, is this issue related to?

e.g. `indentation`

> What CSS is needed to reproduce this issue?

e.g.

```css
$tooltip-default-settings: (
    tooltip-gutter: 8px 10px,
    tooltip-border: 1px solid,
    tooltip-border-radius: 5px,
    tooltip-arrow-border: 8px
);
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "indentation": ["tab"]
  }
}
```

> Which version of stylelint are you using?

 `5.3.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

"PostCSS "

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

"SCSS maps"

> What did you expect to happen?

 "No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

e.g. "The following warnings were flagged:"

```shell
6:1     ‼  Expected indentation of 1 tab (indentation) [stylelint]
```

**Comment by Nirazul** [comment]
I can confirm that. It's also occuring for any indentation with spaces.

**Comment by alan-agius4** [comment]
It something related with the latest release at before it was working just fine

**Comment by davidtheclark** [comment]
I think I know the reason for this, and it sucks. I switched handling of Sass maps so they *weren't* interpreted as functions; but now `indentation` is not ignoring them like it does other functions. There's no way to win with Sass maps :(

**Comment by evilebottnawi** [comment]
@davidtheclark I deeply understand this problem and try to fix

**Comment by evilebottnawi** [comment]
@davidtheclark With regards to this problem, too bad that we can not solve the check indentation inside the function (include sass maps => all inside `(` and `)`)

**Comment by alan-agius4** [comment]
Any chance of having this fixed and maybe do a hotfix as its kinda breaking
a lot of linte in my projects

**Comment by davidtheclark** [comment]
> its kinda breaking
a lot of linte in my projects

You can disable the rule for now. You can see I made a PR to fix this. Just wanted to get buy-in from others before merging and releasing.

**Comment by davidtheclark** [comment]
> With regards to this problem, too bad that we can not solve the check indentation inside the function (include sass maps => all inside ( and ))

@evilebottnawi If you want to tackle the problem of enforcing indentation inside function parameters and Sass maps, feel free to PR. I'm not against it, just am not planning to tackle it myself.

**Comment by evilebottnawi** [comment]
@davidtheclark https://github.com/stylelint/stylelint/issues/892 i try to solved problem on next week :beer:

**Comment by davidtheclark** [comment]
Released in [5.4.0](https://github.com/stylelint/stylelint/releases/tag/5.4.0)

**Comment by alan-agius4** [comment]
Thanks :)

**Comment by Nirazul** [comment]
@evilebottnawi 
Are you still interested in fixing list (and indentation) rules for scss lists within parens?
If yes, I think the best way to handle lists within parens is to just ignore the opening and closing parens and test it against all rules as if it was a normal list. What do you think?
